hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/replication/regionserver/MetricsReplicationSourceSourceImpl.java

```diff
@@ -126,6 +129,9 @@ public MetricsReplicationSourceSourceImpl(MetricsReplicationSourceImpl rms, Stri
 
     oldestWalAgeKey = this.keyPrefix + "oldestWalAge";
     oldestWalAge = rms.getMetricsRegistry().getGauge(oldestWalAgeKey, 0L);
+
+    sourceInitializingKey = this.keyPrefix + "numInitializing";
```

Comment:
nit: I think for a given source, numInitializing doesn't make much sense since its either initializing or not? Is there a better way to do this? I think it is helpful to know which is source is stuck but not sure how better to represent this information.

Reply:
yeah, I think for global level, it should be numInitializing and the source level should be isInitializing